index.html

```diff
@@ -79,5 +79,7 @@ <h4 class="logo">Relaxr</h4>
 
 			<p class="copyright">Copyright 2015. Relaxr.</p>
 		</footer>
+	<script src="http://code.jquery.com/jquery-1.12.0.min.js"></script>
+	<script type="text/javascript" src="jscript/jquery.js"></script>
```

Comment:
I'd recommend not calling your own authored files after a library. This can get confusing. When I originally looked at this, it looked you were including two copies of jQuery. You could call your own file `index.js`, `main.js`, `script.js` or whatever else makes sense.